### ISSUE TITLE
Export to .epub format?

### ISSUE THREAD

**Issue by Thalass** [ligi/SurvivalManual#98]
Apologies if this is already a thing. I have the app on my android device, and it can print to pdf, sections of the book at least. Is there a way to export the whole thing to epub? I have an ebook reader I'd like to view this on. I've tried converting the wiki to epub but haven't had much luck, that's supposed to be possible. 
Thanks

**Comment by rzr** [comment]
Yes this is desirable, a pdf file would be great too something that can be printed and kept safe offline

Maybe this is done in the upstream too ?


I came to this wiki because I have a similar project

https://github.com/rzr/solarpunk-bootstrap/

**Comment by inferno986return** [comment]
The way I see it, a PDF could be typeset (in a program such as InDesign/Affinity Publisher or Scribus) and then an ePub can be made. What I would l ultimately like to see is a physical book that we can put on a shelf as that doesn't require power.

**Comment by Thalass** [comment]
I've been doing some reformatting, though not for a week or two, to book-ify it for my epub copy. I'm pretty sure that no matter what comes first, pdf or epub, the other one will need tweaking for it to display properly. 
Paper would be best, of course, for survival and camping and all that. Especially if it could be printed on some kind of wax paper or something.

**Comment by inferno986return** [comment]
I have made a start at creating a 2.0.1 ePub within my own tools on: https://github.com/inferno986return/SurvivalManual-ebook

There is quite a bit of work to do to make this look nice when printed.

**Comment by inferno986return** [comment]
As a follow-up this issue can probably be closed. I have a working ePub and am making minor alterations to it.

I have made content alterations (most of which I have documented in my conversion notes) and will be implementing them back to the wiki.
